server.js

```diff
@@ -14,10 +17,11 @@ app.get('/',(req,res)=>{
 
 
 
+
 app.get('/ping',(req,res)=>{
     res.json({message:'Pong'})
 })
 
-app.listen(3000,()=>{
-    console.log('Server is working on 3000')
+app.listen(4000,()=>{
```

Comment:
The server port should also be stored in an environment variable, with a default value for development. This makes the application more flexible and ready for deployment. [medium]